main/main.go

```diff
@@ -16,8 +17,9 @@ func init() {
 		IncludeLocation: false,
 		Output:          os.Stderr,
 	}
-}
-
-func main() {
+	if hclog.DefaultOptions.Level <= hclog.Debug {
+		// Tell issue reporting to amend all errors with a stack trace.
+		issue.IncludeStacktrace(true)
+	}
```

Comment:
@scottyw I forgot to mention it, but this if-statement should probably be added to all plug-ins.

Reply:
NP, I can take care of that.